.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,24 @@
+name: pre-commit
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths-ignore:
+      - '*.md'
```

Comment:
Why?

Reply:
Random copy/paste from the existing action we had